release.sh

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env bash
+
+# Copyright © 2022 The Tekton Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+RELEASE_VERSION=""
+
+declare -r SCRIPT_PATH=$(readlink -f "$0")
+declare -r SCRIPT_DIR=$(cd $(dirname "$SCRIPT_PATH") && pwd)
+declare -r API_DIR="$SCRIPT_DIR/api"
+declare -r UI_DIR="$SCRIPT_DIR/ui"
+DOCKER_CMD=${DOCKER_CMD:-docker}
+REGISTRY_BASE_URL=${REGISTRY_BASE_URL:-quay.io/tekton-hub}
+
+declare -a BINARIES=(
+  git
+  hub
+)
```

Comment:
are we using these BINARIES anywhere? Also can you please add a function that can whether these binaries exist, if these binaries do not exist then do not proceed with the release?

Reply:
moreover, we'll be needing `ko` and `make` as well before proceeding so can you please those as well in this list?